pkg/listen/source.go

```diff
@@ -180,6 +180,7 @@ func createSource(sdkClient *hookdeckclient.Client, name *string) (*hookdecksdk.
 		sourceName = answers.Label
 	}
 
+	slug.Lowercase = false
```

Comment:
@alexluong - any chance this has any side effects elsewhere? For example, does it set `Lowercase` globally, and if `slug` is used in some other file/function then the `Lowercase` setting will be set as `true`?

Ultimately, I'm assuming we only really want to use the function in `Lowercase`, but if we do I think we should have a utility function of our own to make it clear that the `Lowercase` is intended.